components/vault/ChartCard.tsx

```diff
@@ -97,7 +98,7 @@ function	ChartCard({address, price, chainID}: {address: string, price: number, c
 					pricePerShare
 				}
 			}
-		}` : null, (query: string): any => request(networks[chainID === 1337 ? 1 : chainID  || 1].graphURI, query)
+		}` : null, (query: string): any => request(networks[chainID === 1337 ? 1 : chainID  || 1].graphURI!, query)
```

Comment:
<img width="717" alt="Screenshot 2023-05-01 at 10 57 53" src="https://user-images.githubusercontent.com/78794805/235425950-2c90ff72-164c-4f8f-b88e-a830d0365b39.png">

Just because `graphURI` might be `undefined`, but I agree with you that it might be a bit risky.

Refactored to not make the request if we don't have the `graphURI` with https://github.com/yearn/macarena-finance/pull/24/commits/ecf9925a41e5da3ad8ac379d10999e47db8f5856